### PR TITLE
Standards apps urls/19

### PR DESCRIPTION
**Pull request by DACunningham**
Moved the standards app's urls to the urls file in the standards app.  It can get a little fiddly with reversing the urls for use in templates and views but it is workable.  I think we will need to change to class based views soon anyway but this is a very useful step in that direction.  This work also needs to be done for the working_solutions app.

### CODE REVIEW

File: lab_log/urls.py [hunk 1]
```diff
@@ -1,22 +1,18 @@
...
-from standards import views as standards
+from django.urls import path, include
+from standards.views import home
 from user_management import views as user_management
 from working_solutions import views as ws
 
 
 urlpatterns = [
+    path("", home, name="home"),
+    path("standards/", include("standards.urls")),
```
Comment (DACunningham) [inline lab_log/urls.py:12]: import the new url file

File: standards/templates/standards/home.html [hunk 2]
```diff
@@ -57,7 +57,7 @@ <h4><a href="{% url 'assay-view' %}?q='rtd'"> Find RTD Standards new</a> </h4>
 
             {% if request.user.is_authenticated %}
             <h3> Log Material </h3>
-                <h4> <a href="{% url 'liquid-create' %}"> Log a Liquid Standard </a> </h4>
+                <h4> <a href="{% url 'standards:liquid-create' %}"> Log a Liquid Standard </a> </h4>
```
Comment (DACunningham) [inline standards/templates/standards/home.html:60]: need to define what app to look in for the url

File: standards/views.py [hunk 3]
```diff
@@ -84,7 +85,7 @@ def liquid_create(request):
 
             new_liquid.save()
             messages.success(request, "Standard successfully logged. Log another...?")
-            return redirect("liquid-create")
+            return redirect(reverse("standards:liquid-create"))
```
Comment (DACunningham) [inline standards/views.py:97]: function that reverses urls.  This is better than using 'magic strings' ('liquid-create') as, in future when we have class based views, we can define the view and not have to worry about the urls at all.

**Review by Marke45 — APPROVED** [review]
Looks like I was at least half way there when I attempted this. In fact I'm not entirely sure why my version didn't work. I'll work on the rest of the urls. Haven't forgotten about the move to class-based views.

**Pull request merged**